checks.xml

```diff
@@ -235,11 +235,6 @@
         </module>
         <module name="OverloadMethodsDeclarationOrder"/>
         <module name="VariableDeclarationUsageDistance"/>
-        <module name="CustomImportOrder">
```

Comment:
Why was this removed?

Reply:
All these checks generate a too huge number of warnings in the log.  
No one actually going to spend his time fixing that, but it makes the log analysis much harder.

I see that it can be useful only when we enable and make this check required.

Also, I have deleted the checkstyle javadoc checks. 
Overall it generated so many warning which prevented travis-ci to build the project.
It exceeds the 10k lines limit in Travis-ci.

Reply:
Please put this into a separate commit.

Reply:
I can't, if I leave it as is I wouldn't be able to build the project.
When we deploy dependencies we also deploy source/javadoc from now. 
And checkstyle started to generate too many warnings, I had to disable those checks.

Reply:
You are claiming that you are unable to build the project -- warnings aside -- what is the problem? If you are running into Checkstyle compliance issues, then the right solution is to fix your code.

Reply:
It's not my code,
It's thousands of thousands of warnings (javadoc + imports) across all projects in the Corfu.

Reply:
The reason why it wasn't broken before is that before we build only jar files, now we also build javadoc+sources for you modules

Reply:
And when you build jar+javadoc+sources it generate 3x more warnings.
I believe having sources+javadoc for your artifact is way more important than having checkstyle warning that everyone ignores

Reply:
Can you please be more specific. What exactly is broken? Does the code not compile?

I checked-out your branch and did `git checkout HEAD~10 ./checks.xml`. I am not seeing any compilation issues.

Reply:
The build on `travis-ci` is broken. 10k log limit.
The code is ok, you can compile it, it passes or the tests.

```
Also, I have deleted the checkstyle javadoc checks.
Overall it generated so many warning which prevented travis-ci to build the project.
It exceeds the 10k lines limit in Travis-ci.
```

Reply:
Looking at the recent Travis builds and test runs, I am not seeing any failures. Can you please point me to specific Travis CI failures that are caused by verbose output? For example, here is one of the recent PRs: https://travis-ci.org/CorfuDB/CorfuDB/builds/600908524

The question about check-style enforcement and log verbosity is something we should look into _outside_ of this patch. I see you are already passing `-q` flag to the Travis pipeline, which reduces log verbosity.

Reply:
Again:
1. Corfu build in my branch generates much more logs, because I build javadoc+sources for the jar artifacts
2. You won't see the issue in other branches, see point 1.
3. this build https://travis-ci.org/CorfuDB/CorfuDB/jobs/597484304

what I did:
1. changed checkstyle
2. added `-q` parameter, to reduce the log https://github.com/CorfuDB/CorfuDB/blob/0ff85b92982ca2ec8f0fd214fbf4e1b3efb30aa7/.travis.yml#L19

Reply:
Also if you disable all the logs what is the point for the warnings?
We just disable everything, it's even more than disable a couple of checks

P.S. yes I understand the diff between checkstyle and warnings in the log, but still

Reply:
As discussed offline, we bring those checks back and double-check that it causes the problem. If not then we don't change check style. If yes, will figure out what to do